WorkspaceServer/Kernel/INativeAssemblyLoadHelper.cs

```diff
@@ -1,11 +1,17 @@
 ﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System;
+using System.Collections.Generic;
+using System.IO;
+using WorkspaceServer.Packaging;
+
 namespace WorkspaceServer.Kernel
 {
-    public interface INativeAssemblyLoadHelper
+    public interface INativeAssemblyLoadHelper : IDisposable
```

Comment:
We made this `IDiposable`, but I don't see the instances being disposed of.

Reply:
It gets registered with the kernel:

https://github.com/dotnet/try/pull/600/files/775503234fee4b7966e947640b1fd1166b8ed347#diff-5b3346e53f8139b99a4f6237e72f4132R102